src/tools/tidy/src/ui_tests.rs

```diff
@@ -7,7 +7,7 @@ use std::path::Path;
 
 const ENTRY_LIMIT: usize = 1000;
 // FIXME: The following limits should be reduced eventually.
-const ROOT_ENTRY_LIMIT: usize = 948;
+const ROOT_ENTRY_LIMIT: usize = 949;
```

Comment:
What is this?^^

Reply:
Tidy shouted at me

Reply:
That constant really needs documentation for what it even means...

Reply:
I think that's the amount of UI tests not in a subdirectory. A number that should be as low as possible :)

Reply:
Yeah @c410-f3r is regularly filing PRs to reduce that number, e.g. #102938